composer.json

```diff
@@ -21,7 +21,7 @@
         "ext-curl": "*",
         "beberlei/assert": "^2.0",
         "symfony/yaml": "~2.1|^3.0|^4.0",
-        "symfony/event-dispatcher": "^2.4|^3.0|^4.0"
+        "symfony/event-dispatcher": "^5.0"
```

Comment:
Is php-vcr 1.5 going to drop [Symfony 3.4 and 4.4, which are LTS releases](https://symfony.com/releases)?

Reply:
I think we should indeed continue to support Symfony 3.4 and 4.4. Thank you for asking that question!